addons/web/static/src/views/view_components/view_scale_button.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="web.ViewScaleButton" owl="1">
+        <span t-if="Object.keys(props.scales).length > 1" class="o_view_scale_button">
```

Comment:
Shouldn't we remove this span ?
It does not serve anything here, except maybe some styling rules, but since we are in master I think we should remove it.
Put the t-if on the Dropdown and also the class.